index.js

```diff
@@ -1608,6 +1608,18 @@
     return Bifunctor.methods.bimap(bifunctor)(f, g);
   }
 
+  //# mapLeft :: Bifunctor f => (a -> b, f a c) -> f b c
+  //.
+  //. Maps the given function over the first type argument of a Bifunctor
+  //.
+  //. ```javascript
+  //. > mapLeft(s => s.toUpperCase(), Tuple('foo', 'bar'))
+  //. Tuple('FOO', 'bar')
+  //. ```
+  function mapLeft(f, bifunctor) {
+    return Bifunctor.methods.bimap(bifunctor)(f, identity);
```

Comment:
I chose to avoid the extra function call to `bimap`. If everyone agrees with this, it might be a good idea to do the same with other derived functions.

Reply:
I'm happy to make this change, but I'd like to do it for all functions at once (either before or after this pull request is merged). I dislike inconsistency, even when it is short-lived.

Reply:
I'd prefer to make that change before this is merged then.

Reply:
:thumbsup:

Reply:
Since #82 is currently on hold, let's use `bimap` here for now so this pull request can be merged. :)

Reply:
⚡️